deploy_nixos/nixos-instantiate.sh

```diff
@@ -2,9 +2,9 @@
 set -euo pipefail
 
 # Args
-nix_path=$1
-config=$2
-config_pwd=$3
+nix_path="$1"
+config="$2"
+config_pwd="$3"
```

Comment:
This commit is not necessary. Bash doesn't expand the value when setting variables.

Reply:
Hah, didn’t know that.